packages/cli/src/module-resolver.ts

```diff
@@ -327,7 +339,14 @@ export function getCommonJsExports(
   system: System,
   parentUrl: string,
 ): string[] {
-  const resolution = resolveModule(packageSpecifier, parentUrl, system);
+  const relative = isRelative(packageSpecifier);
```

Comment:
@Mrtenz If you find the changes here too hacky we can also define a new function for gathering the exports recursively from a given file, but that seemed like it would introduce a lot of duplication.

Reply:
I think this looks fine.